src/imageLoader/wadors/getPixelData.js

```diff
@@ -29,9 +29,144 @@ function uint8ArrayToString(data, offset, length) {
   return str;
 }
 
-function getPixelData(uri, imageId, mediaType = 'application/octet-stream') {
+/**
+ * Asserts that a given media type is valid.
+ *
+ * @params {String} mediaType media type
+ */
+
+function assertMediaTypeIsValid(mediaType) {
+  if (!mediaType) {
+    throw new Error(`Not a valid media type: ${mediaType}`);
+  }
+  const sepIndex = mediaType.indexOf('/');
+
+  if (sepIndex === -1) {
+    throw new Error(`Not a valid media type: ${mediaType}`);
+  }
+  const mediaTypeType = mediaType.slice(0, sepIndex);
+
+  const types = ['application', 'image', 'text', 'video'];
+
+  if (!types.includes(mediaTypeType)) {
+    throw new Error(`Not a valid media type: ${mediaType}`);
+  }
+  if (mediaType.slice(sepIndex + 1).includes('/')) {
+    throw new Error(`Not a valid media type: ${mediaType}`);
+  }
+}
+
+/**
+ * Parses media type and extracts its type and subtype.
+ *
+ * @param mediaType e.g. image/jpeg
+ * @private
+ */
+function parseMediaType(mediaType) {
```

Comment:
Function name is not what it does in my opinion, maybe we just don't use a function since it is two lines of code, and we `assert` and `split` directly?

Reply:
removed in https://github.com/cornerstonejs/cornerstoneWADOImageLoader/pull/477/commits/2174d5ef0bfe8a2d32642110b3d2094255a89889

Reply:
removed in https://github.com/cornerstonejs/cornerstoneWADOImageLoader/pull/477/commits/55628dd0ed4ef2b8d77cd6cb3394f9ba264ad5e6